openapi-cli/src/main/java/io/ballerina/openapi/generators/client/BallerinaAuthConfigGenerator.java

```diff
@@ -131,15 +141,17 @@
     private boolean isHttpOROAuth = false;
     private final Set<String> authTypes = new LinkedHashSet<>();
     private String apiKeyDescription;
+    private final GeneratorUtils generatorUtils;
```

Comment:
Shall we remove adding this util class here and keep the methods static since the function `createParamAPIDoc` behave like the util?

Reply:
yes, but the only usage of  that function is only in this class. What are the drawbacks of creating an `GeneratorUtil` class instance here ?

Reply:
We can keep the consistency of the Util class as it is instead of creating the instance of the class. :)

Reply:
Yeah sure we can do that. The way I have used here is the same way we have done it in the `BallerinaClientGenerator`. Are u suggesting to apply the change to any class that have used `createParamAPIDoc` function ?

Reply:
Yeah, it is better to remove the GeneratorUtil instance usage of `BallerinaClientGenerator` and ` FunctionSignatureGenerator`, Make it as a Util, if it is hard ftm, let's do it in the next fixes.

Reply:
Will address this suggestion in the next PR